### DISCUSSION TITLE
Welcome to vue-keyboard-trap Discussions!

### DISCUSSION THREAD

**Discussion by pdanpdan** [pdanpdan/vue-keyboard-trap#1] [Announcements]
## Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together.

**Comment by teleskop150750** [comment]
Very cool work

**Reply by pdanpdan** [reply]
Thank you

**Comment by teleskop150750** [comment]
I hope your package will help me in creating an aria-grid component

[useGridList](https://react-spectrum.adobe.com/react-aria/useGridList.html#adding-selection-checkboxes)

**Reply by pdanpdan** [reply]
If you have suggestions for improvement please open an issue/PR

**Comment by gjaprakash** [comment]
I got this "Failed to resolve directive: kbd-trap"

**Reply by pdanpdan** [reply]
Where? What are you doing?
I just dropped my 8 magic ball and it'll be some time before I'll get it fixed :)

**Reply by gjaprakash** [reply]
So basically i have problem to implement package i have added pack by import like this import { VueKeyboardTrapDirectivePlugin } from '@pdanpdan/vue-keyboard-trap';

this is my component 
![image](https://github.com/pdanpdan/vue-keyboard-trap/assets/28482866/9017f60d-2ca3-47f0-8401-dce6292b4ede)
how to use this VueKeyboardTrapDirectivePlugin , not used

but i added this v-kbd-trap.roving.vertical.tabinside.escrefocus

**Reply by pdanpdan** [reply]
Looking as the screenshot I would guess you are using Vue2. So follow the instructions here
https://pdanpdan.github.io/vue-keyboard-trap/guide/#include-in-specific-components (Vue2 tab)
